shared/validate/validate_cgo.go

```diff
@@ -0,0 +1,51 @@
+//go:build linux && cgo
+
+package validate
+
+/*
+#include "../../lxd/include/config.h"
+#include "../../lxd/include/memory_utils.h"
+#include "../../lxd/include/mount_utils.h"
+
+static int is_bpf_delegate_option_value_valid(char *option, char *value)
+{
+	__do_close int fs_fd = -EBADF;
+	int ret;
+
+	fs_fd = lxd_fsopen("bpf", FSOPEN_CLOEXEC);
+	if (fs_fd < 0) {
+		return 2;
+	}
+
+	ret = lxd_fsconfig(fs_fd, FSCONFIG_SET_STRING, option, value, 0);
+	if (ret < 0) {
+		return 1;
+	}
+
+	return 0;
+}
+*/
+import "C"
+
+import (
+	"fmt"
+	"unsafe"
+)
+
+// IsBpfDelegateOption validates a BPF Token delegation option.
+func IsBpfDelegateOption(delegateOption string) func(value string) error {
```

Comment:
Go's style guide recommends using upper case acronyms in exported functions, so suggest calling this `IsBPFDelegationOption` - there may be other places in this PR where the same change should occur too.

Reply:
Fixed, thanks!